CHANGELOG.md

```diff
@@ -2054,7 +2054,7 @@ In addition to new features, Rocket saw the following smaller improvements:
 
 ## Core
 
-  * Typo in `Outcome` formatting fixed (Succcess -> Success).
```

Comment:
Found this same line "fixed" in #2353 and really don't know if it should be changed

Reply:
@jinakola Fixed.